admin/views/tabs/dashboard/features.php

```diff
@@ -55,9 +55,13 @@
 				'off' => __( 'Off', 'wordpress-seo' ),
 			),
 			'<strong>' . $feature->name . '</strong>',
-			$feature_help->get_button_html() . $feature_help->get_panel_html()
+			$feature_help->get_button_html() . $feature_help->get_panel_html(),
+			$feature
```

Comment:
not sure why the entire `$feature` object is passed as 5th parameter, as `toggle_switch()` accepts 4 params